src/assets/main.scss

```diff
@@ -48,6 +48,7 @@
   --text-grey: #444444;
   --warm-grey: #767676;
   --footer-grey: #ebebeb;
+  --pastel-blue: #216e93;
```

Comment:
The color provided by Baxter, does not replace primary blue, it's an additional color.